src/tests/__snapshots__/RewardsDialog.test.tsx.snap

```diff
@@ -29,137 +28,72 @@ exports[`Rewards Dialog when there are no rewards displays no rewards dialog 1`]
             <div
-              class="MuiGrid-root-133 MuiGrid-item-135 MuiGrid-grid-xs-12-179 MuiGrid-grid-sm-8-189"
+              class="MuiGrid-root-60 makeStyles-successIconContainer-52 MuiGrid-item-62"
             >
-              <div
-                class="MuiGrid-root-133 makeStyles-moreRewardsSection-65"
-              >
-                <div
-                  class="MuiGrid-root-133 makeStyles-moreRewardsInformation-66 MuiGrid-container-134 MuiGrid-direction-xs-column-137"
-                >
-                  <div
-                    class="MuiGrid-root-133 MuiGrid-item-135"
-                  >
-                    <h6
-                      class="MuiTypography-root-80 makeStyles-titleText-57 MuiTypography-h6-90"
-                    >
-                      Want more rewards ?
-                    </h6>
-                  </div>
-                  <div
-                    class="MuiGrid-root-133 makeStyles-moreRewardsDescription-67 MuiGrid-item-135"
-                  >
-                    <p
-                      class="MuiTypography-root-80 MuiTypography-body2-81"
-                    >
-                      Boost your rewards and support the BadgerDAO ecosystem, by holding and staking Badger tokens.
-                    </p>
-                  </div>
-                  <div
-                    class="MuiGrid-root-133 makeStyles-rewardsGuideLinkContainer-69 MuiGrid-item-135"
-                  >
-                    <a
-                      class="MuiTypography-root-80 MuiLink-root-241 MuiLink-underlineHover-243 makeStyles-rewardsGuide-70 MuiTypography-colorPrimary-103"
-                    >
-                      Rewards User Guide
-                    </a>
-                  </div>
-                  <div
-                    class="MuiGrid-root-133 makeStyles-boostRewards-68 MuiGrid-item-135"
-                  >
-                    <button
-                      class="MuiButtonBase-root-119 MuiButton-root-247 MuiButton-contained-255 MuiButton-containedPrimary-256 MuiButton-fullWidth-270"
-                      tabindex="0"
-                      type="button"
-                    >
-                      <span
-                        class="MuiButton-label-248"
-                      >
-                        Boost My Rewards
-                      </span>
-                      <span
-                        class="MuiTouchRipple-root-276"
-                      />
-                    </button>
-                  </div>
-                </div>
-              </div>
+              <img
+                alt="rewards success icon"
+                src="/assets/icons/rewards-claim-success.svg"
+              />
             </div>
+            <h4
+              class="MuiTypography-root-184 makeStyles-rewardsTitle-53 MuiTypography-h4-192"
+            >
+              Rewards Claimed
+            </h4>
+            <h6
+              class="MuiTypography-root-184 MuiTypography-subtitle2-196"
+            >
```

Comment:
this doesn't look right, this snapshot should be about the available rewards state image.png

Reply:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/38574891/180827427-1be7b9f3-45ec-40cc-aef8-cc90fa8d239c.png">